test/NuGet.Core.Tests/NuGet.Packaging.Test/SigningTests/ClientPolicyContextTests.cs

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions;
+using NuGet.Common;
+using NuGet.Configuration;
+using NuGet.Configuration.Test;
+using NuGet.Packaging.Signing;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace NuGet.Packaging.Test
+{
+    public class ClientPolicyContextTests
```

Comment:
Can you also add test to read from multiple NuGet.config files, if not added already?

Reply:
I can add it. I thought it was not needed since this feature has nothing to do with the logic to compute settings.